src/logic/wallets/e2e-wallet/module.ts

```diff
@@ -0,0 +1,30 @@
+
+const getE2EWalletModule = (): WalletModule => {
+  return {
+    name: WALLET_NAME,
+    type: 'injected',
+    iconSrc: CYPRESS_LOGO,
+    wallet: async (helpers) => {
+      const { createModernProviderInterface } = helpers
+      const provider = new HDWalletProvider({
+        mnemonic: window.Cypress.env('CYPRESS_MNEMONIC'),
```

Comment:
Should I rename the secret to CYPRESS_MNEMONIC?

Reply:
By including the mnemonic in the Cypress env file, won't they be part of the repo?

Reply:
> Should I rename the secret to CYPRESS_MNEMONIC?

Yeah, I think is better to change the name as CYPRESS_ vars can as well be automatically loaded at an [OS level ](https://docs.cypress.io/guides/guides/environment-variables#Option-3-CYPRESS_), so more flexible

> By including the mnemonic in the Cypress env file, won't they be part of the repo?
Not really, you can load it in [several ways](https://docs.cypress.io/guides/guides/environment-variables)

I'm doing it like this in the plugins file

```
module.exports = async (on, config) => {
...
  config.env.CYPRESS_MNEMONIC = process.env.CYPRESS_MNEMONIC

  return config
}
```

I removed all the usages for the E2E variable so if you want to do connected tests then you need to load the CYPRESS_MNEMONIC in your preferred way